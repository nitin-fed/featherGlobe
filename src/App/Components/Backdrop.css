.Backdrop {
  height: 100%;
  width: 100%;
  position: fixed;

  left: 0;
  top: 0;
  background-color: rgba(0, 0, 0, 0.7);
  overflow-x: hidden;
  overflow-y: auto;
}
