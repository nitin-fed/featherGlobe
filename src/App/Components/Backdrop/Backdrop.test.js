import React from 'react'
import Backdrop from './Backdrop'
import renderer from 'react-test-renderer';
import { JestEnvironment } from '@jest/environment';

describe('Backdrop should return', () => {
    test('Return null if show == false', () => {
        const props = {
            show: false
        }
        const component = renderer.create(<Backdrop props={props}/>)
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    })

    it('Return null if show == true', () => {
        const props = {
            show: true
        }
        const component = renderer.create(<Backdrop props={props}  />)
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    })
});