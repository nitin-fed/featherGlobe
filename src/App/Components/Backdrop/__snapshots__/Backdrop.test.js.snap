// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Backdrop should return Return 1`] = `null`;

exports[`Backdrop should return Return null if show == false 1`] = `null`;

exports[`Backdrop should return Return null if show == true 1`] = `null`;
